src/less/components/button.less

```diff
@@ -108,6 +110,7 @@
     &:focus {
         outline: 0;
         box-shadow: 0 0 0 0.1rem @focus-shadow;
+        border-color: @color;
     }
 
     &:hover,
```

Comment:
The box shadow for when the Button is active, should now be 1px, with color #FBDD91.